modules/swagger-codegen/src/main/resources/rust/model.mustache

```diff
@@ -0,0 +1,57 @@
+{{>partial_header}}
+{{#models}}
+{{#model}}
+{{#description}}
+// {{{classname}}} : {{{description}}}
+{{/description}}
+
+#[derive(Debug, Serialize, Deserialize)]
+pub struct {{classname}} {
+{{#vars}}
+  {{#description}}
+  // {{{description}}}
+  {{/description}}
+  #[serde(rename = "{{baseName}}")] {{name}}: {{^required}}Option<{{/required}}{{{datatype}}}{{^required}}>{{/required}}{{#hasMore}},{{/hasMore}}
```

Comment:
Also, the TYPE, if it's another model, isn't available in this scope and should be referenced as `super::MODEL_NAME` (from the parent file)